web_poet/page_inputs/response.py

```diff
@@ -0,0 +1,30 @@
+from typing import Union
+
+import attrs
+
+from web_poet.mixins import SelectableMixin, UrlShortcutsMixin
+from web_poet.page_inputs.browser import BrowserResponse
+from web_poet.page_inputs.http import HttpResponse
+from web_poet.page_inputs.url import ResponseUrl
+
+
+@attrs.define
+class AnyResponse(SelectableMixin, UrlShortcutsMixin):
```

Comment:
Would it make sense to add status as well?

Reply:
Missed that one. Thanks! https://github.com/scrapinghub/web-poet/pull/195/commits/895b4183018fe7b69292ae0f6c8c504ec658b7b0